app/templates/buyers/brief_overview.html

```diff
@@ -229,51 +228,82 @@ <h1 class="govuk-heading-xl">{{ brief.get('title', brief['lotName']) }}</h1>
+                {% for section in sections %}
+                  {% if section.step == step_number %}
+                  <li class="dm-task-list__item">
+                    <span class="dm-task-list__task-name">
+                      <a class="govuk-link"
+                        href="{{ brief_links.brief_link_url('grandparent', section, brief) }}"
+                        aria-describedby="step-{{ step_number }}-{{ section.id }}"
+                      >{{ section.name }}</a>
+                    </span>
+                    {% if sections_status[section.slug] == "done" %}
+                    <strong class="govuk-tag dm-task-list__tag" id="step-{{ step_number }}-{{ section.id }}">Done</strong>
+                    {% elif sections_status[section.slug] == 'to_do' %}
+                    <strong class="govuk-tag govuk-tag--grey dm-task-list__tag" id="step-{{ step_number }}-{{ section.id }}">To do</strong>
+                    {% elif sections_status[section.slug] == 'optional' %}
+                    <strong class="govuk-tag govuk-tag--grey dm-task-list__tag" id="step-{{ step_number }}-{{ section.id }}">Optional</strong>
+                    {% elif sections_status[section.slug] == 'in_progress' %}
+                    <strong class="govuk-tag govuk-tag--blue dm-task-list__tag" id="step-{{ step_number }}-{{ section.id }}">In progress</strong>
                     {% endif %}
-                  {% endfor %}
-                </ul>
-            {% endif %}
-            {% if step.links %}
-              <ul class="instruction-list-item-top">
-                {% for link in step.links %}
-                  {% if not link.allowed_statuses or brief.status in link.allowed_statuses %}
-                    <li>
-                      <a class="govuk-link" href="{{ link.href }}">{{ link.text }}</a>
-                    </li>
+                  </li>
                   {% endif %}
                 {% endfor %}
-              </ul>
-            {% endif %}
+              {% endif %}
+              {% for instruction in step.additional_instructions %}
+                {% if brief.status in instruction.allowed_statuses %}
+                  {% if instruction.startable == true %}
+                  <li class="dm-task-list__item">
+                    <span class="dm-task-list__task-name">
+                      <a class="govuk-link" 
+                        href="{{ instruction.href }}"
+                        aria-describedby="step-{{ step_number }}-link-{{loop.index}}"
+                      >{{ instruction.text }}</a>
+                    </span>
+                    <strong class="govuk-tag govuk-tag--grey dm-task-list__tag" id="step-{{ step_number }}-instruction-{{ loop.index }}">{{ instruction.active_tag }}</strong>
+                  </li>
+                  {% else %}
+                  <li class="dm-task-list__item">
+                    <span>{{ instruction.text }}</span>
```

Comment:
Is there a reason why this `span` doesn't have the `dm-task-list__task-name` class?

Reply:
There IS a reason, and the reason is I missed this...